From 203ff7bc85fdf89fd9635335ceea9bbddd0ed471 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sat, 28 May 2022 23:40:52 +0300
Subject: [PATCH] [SW-0] make: rename preboot max size variable

Rename PREBOOT_MAX_SIZE variable to PREBOOT_BIN_MAX_SIZE.
This is to remove the ambiguity and make it certain that this variable
is the binary maximum size (usually to fit the flash region) and not
something else (e.g. sram section).

Change-Id: I58423cf713e5432c6424a7eb8e490111b697e98c
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 1d8c457..fd9bd02 100644
--- a/Makefile
+++ b/Makefile
@@ -98,8 +98,8 @@
 
 image_validation_preboot: preboot
 	@sz=`stat -Lc "%s" $(PREBOOT_TARGET)`; \
-	if [ $$sz -gt $(PREBOOT_MAX_SIZE) ]; then \
-		echo "$(PREBOOT_TARGET) image size ($$sz) is too big (> $(PREBOOT_MAX_SIZE))"; \
+	if [ $$sz -gt $(PREBOOT_BIN_MAX_SIZE) ]; then \
+		echo "$(PREBOOT_TARGET) image size ($$sz) is too big (> $(PREBOOT_BIN_MAX_SIZE))"; \
 		exit 1; \
 	fi
 
diff --git a/arch/gaudi/arm/Makefile b/arch/gaudi/arm/Makefile
index ea14dba..f4e56e2 100644
--- a/arch/gaudi/arm/Makefile
+++ b/arch/gaudi/arm/Makefile
@@ -1,6 +1,6 @@
 FW_SIZE := 0x4000
 PPBOOT_MAX_SIZE := 20480
-PREBOOT_MAX_SIZE := 131072
+PREBOOT_BIN_MAX_SIZE := 131072
 
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/pll.c)
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/security.c)
diff --git a/arch/gaudi2/arc/Makefile b/arch/gaudi2/arc/Makefile
index 2c95e38..43ddfa0 100644
--- a/arch/gaudi2/arc/Makefile
+++ b/arch/gaudi2/arc/Makefile
@@ -2,7 +2,7 @@
 
 FW_SIZE := 0x8000
 PPBOOT_MAX_SIZE := 16384
-PREBOOT_MAX_SIZE := 130048
+PREBOOT_BIN_MAX_SIZE := 130048
 
 LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports.elf
 
diff --git a/arch/gaudi2/arm/Makefile b/arch/gaudi2/arm/Makefile
index 0e276bb..59e82c8 100644
--- a/arch/gaudi2/arm/Makefile
+++ b/arch/gaudi2/arm/Makefile
@@ -2,7 +2,7 @@
 
 FW_SIZE := 0x5400
 PPBOOT_MAX_SIZE := 20480
-PREBOOT_MAX_SIZE := 130048
+PREBOOT_BIN_MAX_SIZE := 130048
 
 PREBOOT_SRCS_C += $(shell ls $(PREBOOT_ROOT_DIR)/lib/preboot/thermal_v2.c)
 
diff --git a/arch/gaudi3/arc/Makefile b/arch/gaudi3/arc/Makefile
index 2cfbb29..a9ebdaa 100644
--- a/arch/gaudi3/arc/Makefile
+++ b/arch/gaudi3/arc/Makefile
@@ -1,6 +1,6 @@
 FW_SIZE := 0x8000
 PPBOOT_MAX_SIZE := 16384
-PREBOOT_MAX_SIZE := 130048
+PREBOOT_BIN_MAX_SIZE := 130048
 
 image_validation: image_validation_common
 image_validation: image_validation_preboot
diff --git a/arch/greco/arc/Makefile b/arch/greco/arc/Makefile
index 5b02e80..3c97c82 100644
--- a/arch/greco/arc/Makefile
+++ b/arch/greco/arc/Makefile
@@ -1,7 +1,7 @@
 include $(HL_DEV_ARCH_COMMON)/Makefile
 
 FW_SIZE := 0x8000
-PREBOOT_MAX_SIZE := 64512
+PREBOOT_BIN_MAX_SIZE := 64512
 
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/pll.c)
 PREBOOT_SRCS_C += $(shell ls $(PREBOOT_ROOT_DIR)/lib/preboot/thermal_v2.c)
